medusa/clients/torrent/qbittorrent.py

```diff
@@ -2,7 +2,7 @@
 
 """qBittorrent Client."""
 
-from __future__ import unicode_literals
+from __future__ import division, unicode_literals
```

Comment:
I was wrong, `division` is not needed in this case, as it's only for the normal division (`/`) operator, and does not affect modulo (`%`).